packages/component/src/SendBox/UploadButton.js

```diff
@@ -25,6 +25,10 @@ const ROOT_CSS = css({
   }
 });
 
+function disabledListener(event) {
```

Comment:
We should use the same style for 'PREVENT_DEFAULT_HANDLER' to be consistent 

https://github.com/microsoft/BotFramework-WebChat/pull/3150/files#diff-013511e58f42302a34558c03f4cdcdbbR6